menu.py

```diff
@@ -204,11 +208,13 @@ def draw_add_node_operator(layout, nodetype, label=None, icon_name=None, params=
     """
 
     default_context = bpy.app.translations.contexts.default
-    node_rna = getattr(bpy.types, nodetype).bl_rna
+    node_rna = get_node_class_reference(nodetype)
```

Comment:
isn't this missing a `.bl_rna` at the end?

Reply:
Added. But I do not see any difference. It seems node class and it's rna both have the same bl_label property.